README.md

```diff
@@ -50,6 +50,7 @@ Store is used to map an upload's fingerprint with the corresponding upload URL.
 | Name | Backend | Dependencies |
 |:----:|:-------:|:------------:|
 | MemoryStore | In-Memory | None |
+| LeveldbStore | LevelDB | None |
```

Comment:
I mean, when you go get it will get extra repositories. But it could also include other dependencies like Redis DB.